src/DataDefinitionsBundle/Resources/config/services.yml

```diff
@@ -245,6 +245,12 @@ services:
         tags:
             - { name: data_definitions.interpreter, type: twig, form-type: Wvision\Bundle\DataDefinitionsBundle\Form\Type\Interpreter\TwigInterpreterType }
 
+    Wvision\Bundle\DataDefinitionsBundle\Interpreter\MetadataInterpreter:
+        arguments:
+            - '@service_container'
```

Comment:
not needed

Reply:
Indeed, argument there is not used. I have removed it